backend/src/relationships/friendships/entities/friendship.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { Exclude } from 'class-transformer';
+import { User } from 'src/users/entities/user.entity';
+import {
+  Check,
+  Column,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  RelationId,
+  Unique,
+} from 'typeorm';
+
+export enum FriendshipTypeEnum {
+  PENDING = 1,
+  ACCEPTED = 2,
```

Comment:
```suggestion
  PENDING = "pending",
  ACCEPTED = "accepted",
```
I would consider using strings as the Enum value for two reasons:
- it provides a clearer response for the client rather than just receiving a 1 or a 2
- it allows Swagger to function easily by sending a string rather than an integer (currently to perform a PATCH you have to change the status field from `"PENDING"` to `1` otherwise you get an internal service error)

Reply:
changed in next commit